Kickstarter-iOS/DataSources/PaymentMethodsDataSourceTests.swift

```diff
@@ -10,12 +10,35 @@ internal final class PaymentMethodsDataSourceTests: XCTestCase {
   let tableView = UITableView(frame: .zero)
 
   func testDataSource() {
+    let cards = GraphUserCreditCard.template.storedCards.nodes
+    self.dataSource.load(creditCards: cards)
+
+    XCTAssertEqual(7, self.dataSource.tableView(self.tableView, numberOfRowsInSection: 0))
 
+    XCTAssertEqual("CreditCardCell", self.dataSource.reusableId(item: 0, section: 0))
+  }
+
+  func testCardDeletion() {
     let cards = GraphUserCreditCard.template.storedCards.nodes
     self.dataSource.load(creditCards: cards)
 
     XCTAssertEqual(7, self.dataSource.tableView(self.tableView, numberOfRowsInSection: 0))
 
     XCTAssertEqual("CreditCardCell", self.dataSource.reusableId(item: 0, section: 0))
```

Comment:
I don't think we need to assert the Cell value again, since it's already being testes on the `testDataSource` function. Actually, since this code is identical to the test above, we could either put everything in one `testDataSource` test or delete the first assertions.